pandas/_libs/tslibs/period.pyx

```diff
@@ -290,6 +317,20 @@ cdef int dInfoCalc_SetFromAbsDateTime(date_info *dinfo,
     """
     Set the instance's value using the given date and time.
     Assumes GREGORIAN_CALENDAR.
+
+    Parameters
+    ----------
+    dinfo : date_info*
+    absdate : int64_t (as returned from get_python_ordinal or absdate_from_ymd)
```

Comment:
not a great comment (nor does it follow doc-string conventions)

Reply:
Will change.